ontology/yaml/resources/EMERGENCY_MONITORING/entity_types/GM.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# General Types
+GM_STD:
```

Comment:
What is the difference with the Gas meter?
https://github.com/google/digitalbuildings/blob/master/ontology/yaml/resources/METERS/entity_types/GAS_METER.yaml

Reply:
whats the comment? it isnt different from anything previously. are you suggesting a different name @berkoben?

Reply:
yeah, I'm suggesting that maybe the generaltype is GASDET.

Reply:
this is a gas meter, not a gasdet.